docs/source/writing-daml.rst

```diff
@@ -4,6 +4,14 @@
 Writing Daml
 =============
 
+Daml is a smart contract language designed to build composable applications on the `Daml Ledger Model <https://docs.daml.com/concepts/ledger-model/index.html#da-ledgers>`_.
+
+The Writing Daml section will teach you how to write Daml applications that run on any Daml Ledger implementation, including key language features, how they relate to the Daml Ledger Model, and how to use Daml’s developer tools to write, test, compile, package and ship your application. It also covers the structure of a Daml Ledger as it pertains to designing your application.
```

Comment:
Not something that needs to be handled here, but I think we should standardize on whether we want to capitalize the word "ledger" or not and in what circumstances.

Reply:
A good question. I was trying to treat "Daml Ledger" as a single proper noun, but there may be cases where that isn't quite right.